README.org

```diff
@@ -116,5 +108,10 @@ Copyright © 2013 - 2016 to the anubad Authors.
   If you wish to contribute this project, there is much thing you can
   do beyond programming, to get started read [[https://github.com/foss-np/anubad/tree/experimental/CONTRIBUTING.org][this]].
 
-  Any code Contributions to anubad should be made via of GitHub pull
+  Any code Contributions to anubad should be made via GitHub pull
   requests.
+
+[[http://i.creativecommons.org/l/by/3.0/88x31.png]]
+
+Copyright © 2013 - 2016 to the anubad Authors.
```

Comment:
@rhoit I usually break down line at 80 char; and I tried to follow the your `MarkDown` practice. I think we follow a lot different conventions in MarkDown. It was kind of :rocket: science for me to get used to your convention :laughing: